### DISCUSSION TITLE
A simple project don't work as expected, please help.

### DISCUSSION THREAD

**Discussion by cnshsliu** [parcel-bundler/parcel#6555] [Q&A]
# Need Help

I am new to Parcel, there is something wrong with my simple project, just don't know what to do next, kindly help.thanks.

## 🎛 Configuration (.babelrc, package.json, cli command)

there is no .babelrc, 
package.json should be found in github repository: 
https://github.com/cnshsliu/dtme
Thanks

```js
{
  "your": { "config": "here" }
}
```

## 🤔 Expected Behavior

run "npm start" should start the development server 

## 😯 Current Behavior

```
Server running at http://localhost:1234
🚨 Build failed.
Error: The "path" argument must be of type string. Received undefined
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
```

## 🔦 Context

I just started a new project with parcel@next, there are one "index.html" and one "index.js" within src folder.
very simple to start, I am new to Parcel, dont' know what to do next.

## 💻 Code Sample

The source code is here: https://github.com/cnshsliu/dtme

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-beta.3.1
| Node             | v14.17.3
| npm/Yarn         | 6.14.13
| Operating System | macOS Big Sur 11.4

**Comment by DeMoorJasper** [comment]
Your example works fine for me, maybe try re-installing dependencies or clearing the cache?
